bot.rb

```diff
@@ -0,0 +1,68 @@
+require 'dotenv/load'
+require 'nokogiri'
+require 'telegram/bot'
+require 'open-uri'
+require 'logger'
+
+TOKEN = ENV['TG_TOKEN'].freeze
+URL = 'http://bash.im/best'.freeze
+$posts = []
+
+# Parse method
+def parse_posts
+  page = Nokogiri::HTML(open(URL))
+  page.css('.quote').css('.text').each do |a|
+    post = a.text
+    $posts.push(post)
+  end
+end
+
+parse_posts
+
+# Bot runner
+Telegram::Bot::Client.run(TOKEN) do |bot|
+  begin
+    bot.listen do |message|
+      case message.text
+      when '/posts'
+        bot.api.send_message(
+          chat_id: message.chat.id,
+          text:  "Hi, #{message.from.first_name}.\n
+          Total posts: #{$posts.length}"
+        )
+        bot.api.send_message(
+          chat_id: message.chat.id,
+          # don't get how to insert '/n' to this yet
+          text: $posts.first(10).join("\n \n").to_s
+        )
+        # machinegun
```

Comment:
don't leave such things in code. if this code doesn't work and you don't need it - use gist.github.com :)